bcda/responseutils/writer.go

```diff
@@ -117,10 +117,26 @@ func CreateCapabilityStatement(reldate time.Time, relversion, baseurl string) *f
 			},
 		},
 	}
-
+	checkEnv(statement, baseurl)
 	return statement
 }
 
+func checkEnv(statement *fhirmodels.CapabilityStatement, baseUrl string) {
```

Comment:
Any chance we can rename this function to something like `addPatientEndpointToStatement`?  The main purpose of the function is to add the patient endpoint to the capability statement; checking the env var is just something it does under the covers.

Reply:
Yeah I see what you're saying but I avoided calling it those names because theoretically this method is only temporary. Once we "release" the Patient endpoint to all of our environments this function won't even exist and it'll be added the same way just like our other endpoints.